neo/Network/P2P/TaskManager.cs

```diff
@@ -41,8 +44,9 @@ private class Timer { }
         private readonly ICancelable timer = Context.System.Scheduler.ScheduleTellRepeatedlyCancelable(TimerInterval, TimerInterval, Context.Self, new Timer(), ActorRefs.NoSender);
 
         private readonly UInt256 HeaderTaskHash = UInt256.Zero;
+        private readonly UInt256 StateRootTaskHash = Blockchain.GenesisBlock.Hash;
         private bool HasHeaderTask => globalTasks.ContainsKey(HeaderTaskHash);
-
+        private bool HashStateRootTask => globalTasks.ContainsKey(StateRootTaskHash);
```

Comment:
If we use the same hash it could collide with other tasks?

Reply:
only one root task exists, like header task.